### ISSUE TITLE
Add allele information to postprocessing?

### ISSUE THREAD

**Issue by sdwfrost** [spond/IgSCUEAL#12]
The reference alignments only contain the *01 alleles for the V genes. Should there be an additional postprocessing step to compare the query sequence to the different allelic forms, if the family and gene is identified?

**Comment by spond** [comment]
Yes, that would be desirable, but probably in the **want** (vs **need**) category for rearrangement mapping for a single host, since, presumably, only a single allelic form will be present.
